src/FSharpVSPowerTools.Core/Utils.fs

```diff
@@ -3,6 +3,24 @@
 open System
 open System.Diagnostics
 
+
+[<AutoOpen>]
+module Prelude =
+    /// obj.ReferenceEquals
+    let inline (==) a b = obj.ReferenceEquals(a, b)
+    /// LanguagePrimitives.PhysicalEquality
+    let inline (===) a b = LanguagePrimitives.PhysicalEquality a b
+    let inline debug msg = Printf.kprintf Debug.WriteLine msg
+    let inline fail msg = Printf.kprintf Debug.Fail msg
+    let inline isNull v = match v with | null -> true | _ -> false
+    let inline isNotNull v = v |> (not << isNull)
+    let inline dispose (disposable:#IDisposable) = disposable.Dispose ()
+    let inline (+=) (a1:'a byref) (a2:'a) = a1 <- a1 + a2
+    let inline (-=) (a1:'a byref) (a2:'a) = a1 <- a1 - a2
+
+    type 'a List with
+        member x.Last = x.[x.Length-1]
```

Comment:
I ran the tests a few more times and I got results like yous, I don't know what that first set was about :confused: